plugins/services/src/js/containers/services/ServicesTable.js

```diff
@@ -193,6 +188,16 @@ class ServicesTable extends React.Component {
       }
     ];
 
+    if (!isPod) {
+      dropdownItems.splice(1, 0, {
```

Comment:
Let's push the items separately so that we don't need to use splice to add the restart item.